scrapers/deviante.yml

```diff
@@ -0,0 +1,33 @@
+name: "Deviante"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - deviante.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1/text()
+      Date:
+        selector: //h1/preceding-sibling::h2/text()
+        postProcess:
+          - parseDate: January 2, 2006
+      Details: //h2[text()="Description"]/following-sibling::div/p/text()
+      URL: //link[@rel="canonical"]/@href
+      Tags:
+        Name: //a[contains(@href,"/videos/tags")]/text()
+      Performers:
+        Name: //h2//a[contains(@href,"/model/")]/text()
+      Studio:
+        Name: 
+          selector: //link[@rel="canonical"]/@href
+          postProcess:
+            - replace: 
+              - regex: .*www\.
+                with: ""
+              - regex: \.com.+
+                with: ""
```

Comment:
I'm guessing this was mainly used in the subsites scraper? Here its only deviante so a `fixed` name should be used